plugins/modules/packaging/os/apt_rpm.py

```diff
@@ -150,7 +150,7 @@ def main():
         argument_spec=dict(
             state=dict(type='str', default='installed', choices=['absent', 'installed', 'present', 'removed']),
             update_cache=dict(type='bool', default=False, aliases=['update-cache']),
-            package=dict(type='str', required=True, aliases=['name', 'pkg']),
+            package=dict(type='list', required=True, aliases=['name', 'pkg']),
```

Comment:
```suggestion
            package=dict(type='list', elements='str', required=True, aliases=['name', 'pkg']),
```
You also need to update the documentation of the option; add `type: list` and `elements: str` to https://github.com/ansible-collections/community.general/blob/02011c7423c3fa54093459958c5ae3b6c13e0e3e/plugins/modules/packaging/os/apt_rpm.py#L24-L27

Reply:
Thanks for the tips. I am a newbie and I don’t have everything right yet

Reply:
You are doing well :)